### PR TITLE
Use standalone node exporter chart

### PR DESCRIPTION
**Pull request by chrissng**
c.f. https://github.com/prometheus-community/helm-charts/pull/1030

### CODE REVIEW

File: templates/node_exporter.yaml [hunk 1]
```diff
@@ -1,140 +1,156 @@
...
-    # requests:
-    #   cpu: 100m
-    #   memory: 30Mi
-
-  ## Security context to be added to node-exporter pods
-  ##
-  securityContext: ${security_context}
-    # runAsUser: 0
-
-  service:
```
Comment (chrissng) [inline templates/node_exporter.yaml:109]: bunch of these values are not supported anymore but i dont think we use them

File: templates/node_exporter.yaml [hunk 2]
```diff
@@ -1,140 +1,156 @@
...
-  ##
-  enabled: ${enable}
-
-  ## If true, node-exporter pods share the host network namespace
-  ##
-  hostNetwork: ${host_network}
-
-  ## If true, node-exporter pods share the host PID namespace
-  ##
-  hostPID: ${host_pid}
```
Comment (chrissng) [inline templates/node_exporter.yaml:12]: this is now always true if hostNetwork is true

File: templates/node_exporter.yaml [hunk 3]
```diff
@@ -1,140 +1,156 @@
...
-  ## Annotations to be added to node-exporter pods
-  ##
-  podAnnotations: ${annotations}
-
-  ## Labels to be added to node-exporter pods
-  ##
-  pod:
-    labels: ${labels}
-
-  podDisruptionBudget:
```
Comment (chrissng) [inline templates/node_exporter.yaml:89]: pdb is not supported

**Comment by lawliet89** [comment]
For that chart, you can use `nindent`, which is nicer: https://helm.sh/docs/chart_template_guide/function_list/#nindent

**Review by lawliet89 — APPROVED** [review]

**Pull request merged**